se_feedback/sprint-5/feedback_sprint_5_all.md

```diff
@@ -0,0 +1,97 @@
+|      |            |
+|------|------------|
+|Group | Rubber Duck Debuggers |
+|Assignment|Sprint 5|
+|Date|29/05/16|
+|TA|Bastiaan Reijm|
+
+#Sprint Feedback
+Feedback and Grades for Sprint 5.
+
+Total: **9.55**
+
+| User Story | Score |
+|------------|-------|
+| definition |  9   |
+| splitting  |  10     |
+| responsibility | 10  |
+
+| Learning from History | Score |
+|-----------------------|-------|
+| estimation            | 10    |
+| prioritisation        | 10    |
+| reflection            | 9   |
+
+## Notes
+* Well done
+* That Matthias took over Robin's task should be added in a note, it looks like both were assigned responsibility and then one member was removed
```

Comment:
Dear Bastiaan,

We as a team don't think it is useful to add a new team member to this highly functioning team this late in the project. We also don't think it is fair to the other students to allow a student to start with the project this late (because that is, from our point of view, the only reason why this person would be added to this team). Therefore we would like to request the removal of this 'Matthias' person from our team. In our opinion this would be best for all.

Sincerely,

Mathias